### DISCUSSION TITLE
LD Coeffs

### DISCUSSION THREAD

**Discussion by mrdecesare** [phoebe-project/phoebe2#882] [Questions]
Hello!

I am having a difficult time with figuring out my ld coeffs. I have a kepler eb with approx 2 sun sized stars and the way i currently have my code set up, i need ld coeffs however if i can skip them that would be great too, im looking for t0_supconj, my advisor has given me the go ahead to skip ld if i have them thats fine too. I've tried a bunch of different things from the tutorials/workshops/discussions and nothing has worked.  My current code and dataset im working with can be found [here](https://github.com/mrdecesare/phoebe_codes/tree/main).

Anything helps!

**Comment by aprsa** [comment]
If you have two Sun-like stars, then -- in general at least -- you do not have to worry about limb darkening coefficients. If you leave ld_mode='interp' (the default), the code will automatically interpolate specific intensities and account for limb darkening on the fly. In most circumstances that is what you want because the LD treatment will be consistent with the emergent intensity treatment.

Getting t0_supconj is better done outside of phoebe; although we have some support for it by using estimators, more sophisticated tools exist that will in general do a better job at determining t0. The rule of thumb, in general, is to have ephemeral parameters available alongside the dataset(s).

**Reply by mrdecesare** [reply]
Thank you for your response! do you have any information about those other options?

**Reply by aprsa** [reply]
That really depends on the morphology of the light curve (or any other observable); if you google for period finders, it should give you a good start. I like [BLS](https://docs.astropy.org/en/stable/timeseries/bls.html); you can take a look at the [vartools](https://www.astro.princeton.edu/~jhartman/vartools.html) package.

**Reply by mrdecesare** [reply]
Thank you!
